pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+    "hatch-vcs",
+    "hatchling",
+]
+
+[project]
+dependencies = [
+    "array_api_strict",
+    "jax",
+    "numpy",
+    "pytest",
+    "pytest-benchmark",
+]
+description = "Benchmarks for glass-dev/glass"
+dynamic = [
+    "version",
+]
+maintainers = [
+    {email = "n.tessore@ucl.ac.uk", name = "Nicolas Tessore"},
+]
+name = "glass-benchmarks"
+readme = "README.md"
+requires-python = ">=3.10"
+license.text = "MIT License"
+
+[tool.hatch]
+build.hooks.vcs.version-file = "tests/_version.py"
```

Comment:
Does this make sense? Usually you version your package, not tests

Reply:
Already been done but this was the other thing to remove along with `build-system`